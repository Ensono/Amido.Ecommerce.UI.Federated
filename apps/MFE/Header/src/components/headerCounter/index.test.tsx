import {render, waitFor} from '@testing-library/react'

import {HeaderCounter} from '.'

describe('headerCounter', () => {
  it('matches the snapshot', async () => {
    const {container} = render(<HeaderCounter count={0} increment={jest.fn()} decrement={jest.fn()} />)
    await waitFor(() => {
      expect(container).toMatchSnapshot()
    })
  })
})
