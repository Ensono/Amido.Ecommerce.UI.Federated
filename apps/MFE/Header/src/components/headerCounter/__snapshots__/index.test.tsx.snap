// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`headerCounter matches the snapshot 1`] = `
<div>
  <div
    class="header_counter"
  >
    <h2>
      Header Count: 
      <span>
        0
      </span>
    </h2>
    <button>
      Make number go up plz
    </button>
    <button>
      Make number go down pretty plz
    </button>
  </div>
</div>
`;
